app/lib/redis_configuration.rb

```diff
@@ -39,12 +39,38 @@ def namespace
   end
 
   def url
-    ENV['REDIS_URL']
+    if ENV['REDIS_SENTINEL']
+      m = ENV.fetch('REDIS_SENTINEL_MASTER', 'mymaster')
+      p = ENV.fetch('REDIS_PASSWORD', '')
+      "redis://:#{p}@#{m}"
+    else
```

Comment:
Can this just be a `REDIS_SENTINEL_URL` ?

Reply:
@ThisIsMissEm Maybe, although it might also be a list of IPs. So unsure.

Reply:
I guess this is a way to give a password when connecting to the sentinel, right?

I have to admit, I am thoroughly confused by the documentation here: https://github.com/redis/redis-rb/tree/v5.2.0#sentinel-support

It mentions you need to supply different credentials for the redis and the sentinel connection. Using a URL could be a way to circumvent this and use the same credentials for both, but the docs are not really clear.

Do we need to support having different credentials for redis and sentinel?

Also: This looks to me to only support password-only auth. Should we not also support giving a user name?

(Sorry if this is all totally obvious, I had never heard of redis sentinel until a few days ago.)